### PR TITLE
transform back substituted characters

### PR DESCRIPTION
**Pull request by ggreif**
This tries to repair the E2E workflow, by reversing character substitutions that are suggested in the README.

### CODE REVIEW

**Comment by ggreif** [comment]
@svenstaro the CI E2E seems to fail, no idea why...

**Comment by ggreif** [comment]
@svenstaro do you think it would be reasonable to merge this and see if `master` passes CI?

File: src/main.ts [hunk 1]
```diff
@@ -142,7 +142,11 @@ async function run(): Promise<void> {
     const overwrite = core.getInput('overwrite') == 'true' ? true : false
     const prerelease = core.getInput('prerelease') == 'true' ? true : false
     const release_name = core.getInput('release_name')
-    const body = core.getInput('body')
+    const body = core
+      .getInput('body')
+      .replace(/%25/g, '%')
```
Comment (ggreif) [inline src/main.ts:147]: maybe do this last

Comment (ggreif) [reply src/main.ts]: done in 0e6c758.

**Comment by svenstaro** [comment]
> @svenstaro do you think it would be reasonable to merge this and see if `master` passes CI?

Sure let's see what happens.

**Comment by svenstaro** [comment]
Seems good, thanks!

**Pull request merged**